pyproject.toml

```diff
@@ -15,7 +15,7 @@ classifiers = [
         'Topic :: Scientific/Engineering :: Artificial Intelligence',
 ]
 keywords = ['deepecho', 'DeepEcho']
-dynamic = ["version"]
+version = '0.5.1.dev0'
```

Comment:
Let's follow what we did with Copulas:
- https://github.com/sdv-dev/Copulas/pull/380/files#r1513096694

Reply:
Yes I was not sure, I checked with @amontanez24. Done in [b1fa699](https://github.com/sdv-dev/DeepEcho/pull/90/commits/b1fa6997cb79ea650de9656f4258a35d72c8ff9b)